src/test/resources/features/UKDrivingLicence/DLUnrecoverableError.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Driving License Test
+
+ Background: @happy_path @localhost8085
+    Given I navigate to the IPV Core Stub
+    When I navigate to User for Driving Licence CRI dev Page
+    Then I search for Driving Licence user number 5 in the Experian table
+    And I click Go to Driving Licence CRI dev button
+
+  Scenario: Check the Unrecoverable error page
```

Comment:
What triggers this error?

Reply:
added a method to delete a cookie to get the error page.